cds_ils/migrator/default_records.py

```diff
@@ -102,6 +103,27 @@ def create_design_report():
     )
 
 
+def create_unknown_item():
+    """Create unknown item."""
+    int_loc_pid_value = get_internal_location_by_legacy_recid(3).pid.pid_value
```

Comment:
can you please name it:
```suggestion
    MEANINGFUL_NAMING_RECID = 3
    int_loc_pid_value = get_internal_location_by_legacy_recid(MEANINGFUL_NAMING_RECID).pid.pid_value
```